code/modules/admin/holder2.dm

```diff
@@ -88,3 +88,75 @@ you will have to do something like if(client.holder.rights & R_ADMIN) yourself.
 		cdel(holder)
 		holder = null
 	return 1
+
+/client/proc/readmin()
+	if(config.admin_legacy_system)
```

Comment:
This entire proc is quite terrible, take a look at how we do it in tg

Reply:
TG has a different DB backend and admin handler. porting that might take a while™